package.json

```diff
@@ -117,22 +117,22 @@
     "cross-env": "^6.0.3",
     "css-loader": "^1.0.0",
     "cypress": "^8.3.1",
-    "eslint": "^7.7.0",
+    "eslint": "^8.57.1",
     "eslint-config-prettier": "^6.7.0",
-    "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-jsx": "^0.1.0",
     "eslint-plugin-node": "^10.0.0",
     "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "^7.17.0",
-    "eslint-plugin-react-hooks": "^4.1.2",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-react": "^7.37.1",
+    "eslint-plugin-react-hooks": "^4.6.2",
     "express": "^4.15.4",
     "file-loader": "^2.0.0",
     "fork-ts-checker-notifier-webpack-plugin": "^3.0.0",
     "fork-ts-checker-webpack-plugin": "^5.0.14",
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^4.5.2",
     "husky": "^0.14.3",
-    "jest": "^25.2.4",
+    "jest": "^26.6.3",
```

Comment:
v25 has a peer dependency of typescript <v4, so we best update the version 

however >v27, we run into a host of problems with jestdom and timers that I was unable to resolve

after v27, the jestdom environment is no longer included in the main jest package, and must be imported separately

as soon as this happens, many tests start throwing errors that `setImmediate` is undefined

there's a lengthy thread here https://github.com/prisma/prisma/issues/8558 with several workarounds

however, of the solutions I've tried, I did not get a successful test run

`global.setImmediate = jest.useRealTimers` caused tests to hang until they timed out
```
import { setImmediate } from 'timers;
global.setImmediate = setImmediate
```

caused any test that relied on RXJS to throw an error that `bind` was undefined

Reply:
If we'd want to invest in the build setup for browser I think it'd make sense to go to vite & vitest, but I don't think it's worth the time